full-stack-tests/presentation/src/learning-snippets/CommonRules.test.ts

```diff
@@ -0,0 +1,519 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { expect } from "chai";
+import { IModelConnection, SnapshotConnection } from "@itwin/core-frontend";
+import {
+  ChildNodeSpecificationTypes, ContentSpecificationTypes, GroupingSpecificationTypes, KeySet, RelationshipDirection, Ruleset, RuleTypes,
+  StandardNodeTypes,
+} from "@itwin/presentation-common";
+import { Presentation } from "@itwin/presentation-frontend";
+import { initialize, terminate } from "../IntegrationTests";
+import { getFieldByLabel } from "../Utils";
+
+describe("Learning Snippets", () => {
+
+  let imodel: IModelConnection;
+
+  beforeEach(async () => {
+    await initialize();
+    imodel = await SnapshotConnection.openFile("assets/datasets/Properties_60InstancesWithUrl2.ibim");
+  });
+
+  afterEach(async () => {
+    await imodel.close();
+    await terminate();
+  });
+
+  describe("MultiSchemaClasses", () => {
+
+    it("uses all attributes", async () => {
+      // __PUBLISH_EXTRACT_START__ MultiSchemaClasses.Ruleset
+      // This specification selects instances of `bis.PhysicalModel` and `bis.SpatialCategory` classes.
+      // Classes that derive from classes specified in `classNames` this list will not be included due to
+      // `arePolymorphic` attribute being set to `false`.
```

Comment:
```suggestion
      // This ruleset produces content for instances of `bis.PhysicalModel` and `bis.SpatialCategory` classes.
      // Descendants of these classes will be considered incompatible with the specified class filter because
      // `arePolymorphic` attribute is set to`false`.
```